source/game.gamescriptexpression.bmx

```diff
@@ -304,9 +305,9 @@ Function SEFN_programmedata:SToken(params:STokenGroup Var, context:SScriptExpres
 	EndIf
 	
 	Select propertyName
-		Case "cast"                    Return _EvaluateProgrammeDataCast(data, params, tokenOffset)
+		Case "cast"                    Return _EvaluateProgrammeDataCast(data, params, 1 + tokenOffset)
```

Comment:
Statt mit den Offsets rumzurechnen, fände ich es ja schöner einfach eine SubTokenGroup weiterzugeben (params.StartingWithOffset(x)), aber vermutlich ist das performancemäßig unschön.

Reply:
Naja, "Structs" sind an sich relativ kostengünstig (aufm Stack). Und ja, SubTokenGroup schwirrt mir auch schon im Kopf rum (kann ja in einem eigenen PR angegangen werden)